public/app/features/explore/Logs.tsx

```diff
@@ -396,6 +417,7 @@ class UnthemedLogs extends PureComponent<Props, State> {
           </div>
           <LogsNavigation
             logsSortOrder={logsSortOrder}
+            defaultLogsSortOrder={this.getDefaultLogsSortOrder()}
```

Comment:
Having 2 logsSortOrder seems confusing. Can't we just change initial `logsSortOrder` if the query had `FORWARD` direction?

Reply:
I agree however the problem is the fact that the default being `FORWARD` changes the whole behaviour of the component. So just passing in the current sort order leaves the component with no clue about how to do pagination.

Reply:
I think that there is a big chance that we will implement this also for other logging data sources. @ifrost What do you think about adding something as `hasQueryDirectionSupport` similarly to https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/query.ts#L100 and if data source implements it, we would set initial `logsSortOrder` based on used order?